metron-platform/metron-solr/src/test/java/org/apache/metron/solr/integration/SolrUpdateIntegrationTest.java

```diff
@@ -20,15 +20,21 @@
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
+import java.util.Optional;
+
 import org.apache.metron.indexing.dao.IndexDao;
+import org.apache.metron.indexing.dao.MultiIndexDao;
 import org.apache.metron.indexing.dao.UpdateIntegrationTest;
+import org.apache.metron.indexing.dao.update.Document;
 import org.apache.metron.integration.InMemoryComponent;
 import org.apache.metron.solr.dao.SolrDao;
 import org.apache.metron.solr.integration.components.SolrComponent;
+import org.junit.Assert;
+import org.junit.Test;
 
 public class SolrUpdateIntegrationTest extends UpdateIntegrationTest {
 
-  SolrComponent solrComponent;
+  protected static SolrComponent solrComponent;
```

Comment:
Why `protected static`?

Reply:
When multiple tests exist in both SolrUpdateIntegrationTest and UpdateIntegrationTest, solrComponent isn't populated across tests because of how we're attempting to reuse it.  Making it static fixes the issue and matches the pattern in UpdateIntegrationTest.